packages/cozy-doctypes/src/contacts/Contact.js

```diff
@@ -0,0 +1,37 @@
+const getPrimaryOrFirst = property => obj => {
+  if (!obj[property] || obj[property].length === 0) return ''
+  return obj[property].find(property => property.primary) || obj[property][0]
+}
+
+class Contact extends Document {
+  static getInitials(name) {
```

Comment:
should we take a contact as the argument instead of the name?

Reply:
Can you add a jsdoc here?

Reply:
I thought about taking a contact as argument but the code looks a bit odd. Maybe we should remove the string case from here since it is not related to contacts at all but it will leave some sort of `getInitials` function in drive...